src/protocol.h

```diff
@@ -354,28 +354,43 @@ static inline bool MayHaveUsefulAddressDB(ServiceFlags services)
     return (services & NODE_NETWORK) || (services & NODE_NETWORK_LIMITED);
 }
 
+enum class AddrSerialization {
+    DISK,
+
+    // The only time we serialize a CAddress object without nTime is in
+    // the initial VERSION messages which contain two CAddress records.
+    // At that point, the serialization version is INIT_PROTO_VERSION.
```

Comment:
This sentence can be removed now. It's meaningless without a serialization version.

Reply:
Arguably there still is a serialization version, but the link between that and CAddress serialization is purely historical. Removed by incorporating @MarcoFalke's changes below.